src/IO/WriteHelpers.h

```diff
@@ -380,6 +380,162 @@ void writeAnyEscapedString(const char * begin, const char * end, WriteBuffer & b
     }
 }
 
+template <char quote_character, bool escape_quote_with_quote = false, bool escape_backslash_with_backslash = true>
+void writeAnyMarkdownEscapedString(const char * begin, const char * end, WriteBuffer & buf)
+{
+    const char * pos = begin;
+    while (true)
+    {
+        const char * next_pos = find_first_symbols_markdown<
```

Comment:
I have seen some ambiguity in the internet about which characters are considered "special characters" in markdown. Better implement what is specified by a standardized variant, e.g. CommonMark ([here](https://spec.commonmark.org/0.30/#example-12)) \+ also mention in the source code which standard is targeted.